jetty-server/src/main/config/modules/customrequestlog.mod

```diff
@@ -1,43 +1,11 @@
 DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
 
 [description]
-An implementation of requestlog using CustomRequestLog and AsyncRequestLogWriter
-
-[provides]
-requestlog-impl
+Deprecated name for requestlog using custom request logger
```

Comment:
Remove "using custom request logger". Here we only need to know it's a deprecated module.